keyboards/handwired/scottokeebs/scottocmd/readme.md

```diff
@@ -0,0 +1,27 @@
+# ScottoCMD
+
+![ScottoCMD](https://i.imgur.com/rxiMZNnh.jpg)
+
+A 40-key ortholinear keyboard with a 6.25u spacebar, 4 dedicated function keys and 0.96" 128x64 OLED display. Case files available [here](https://github.com/joe-scotto/scottokeebs).
+
+*   Keyboard Maintainer: [Joe Scotto](https://github.com/joe-scotto)
+*   Hardware Supported: ATmega32U4
```

Comment:
Hey Joe,
Would you be able to append this line (8) with either the make and model of OLED display used, or words to the effect of *0.96" 128x64 OLED display with I2C protocol*

Reply:
Good call, I also updated the ScottoGame and Scotto36 to include the OLED hardware.